mirrord/kube/src/api/kubernetes.rs

```diff
@@ -136,22 +136,22 @@ impl AgentManagment for KubernetesAPI {
     }
 }
 
-pub async fn create_kube_api(config: &LayerConfig) -> Result<Client> {
+pub async fn create_kube_api(config: &Option<LayerConfig>) -> Result<Client> {
```

Comment:
I think better design here will be to pass `accept_invalid_certs` and `connect_agent_name` instead of config.

Reply:
i think LayerConfig should be an option, it kinda makes it verbose that we don't want the k8s API to use any of our config data when its None

Reply:
yeah agree on that, will change that